### PR TITLE
[UX] Add Tab Title

### PR DESCRIPTION
**Pull request by NobreHD**
Like the title says it, just adding a title to the tab so it doesn't show a URL
image
to
image

### CODE REVIEW

**Comment by NobreHD** [comment]
Untested code (I don't want to set up everything locally), so please check it before pushing to prod

**Comment by GStudiosX2** [comment]
> Like the title says it, just adding a title to the tab so it doesn't show a URL
> image
> to
> image

```
/root/projects/lyntr/src/routes/MainPage.svelte:292:0 </svelte:head> attempted to close an element that was not open
/root/projects/lyntr/src/routes/MainPage.svelte:292:0
290 |      {/if}
 291 |    {:else}
 292 |  </svelte:head>
        ^
```

File: src/routes/MainPage.svelte [hunk 1]
```diff
@@ -360,3 +360,13 @@
...
 </div>
+
+<svelte:head>
+	{#if page === 'home'}
+		{#if selectedLynt}
+			<title>{selectedLynt.username} on Lyntr: "{selectedLynt.content}"</title>
+		{:else}
+			<title>Lyntr</title>
+		{/if}
+	{:else}
```
Comment (GStudiosX2) [inline src/routes/MainPage.svelte]: This should be `{/if}`

**Review by GStudiosX2 — CHANGES_REQUESTED** [review]

**Comment by GStudiosX2** [comment]
Ah you fixed it already lol

**Comment by GStudiosX2** [comment]
Seems to work if this gets merged I'll have to add it to the messages page too depending which one gets merged first either me or you have to do it

**Comment by face-hh** [comment]
LGTM

**Pull request merged**